test/cloudwatchlogs/publish_logs_test.go

```diff
@@ -332,3 +497,154 @@ func checkData(t *testing.T, start time.Time, lineCount int) {
 	)
 	assert.NoError(t, err)
 }
+
+func ValidateEntity(t *testing.T, logGroup, logStream string, begin, end *time.Time, expectedEntity expectedEntity) {
+	log.Printf("Validating entity for log group: %s, stream: %s", logGroup, logStream)
+
+	logGroupInfo, err := getLogGroup(logGroup)
```

Comment:
You can just use `IsLogGroupExists` from our common package: https://github.com/aws/amazon-cloudwatch-agent-test/blob/main/util/awsservice/cloudwatchlogs.go#L137

The helper functions that I wrote was because we had to create a new client in us-east-1 for beta purpose but they are no longer needed